tracing-appender/src/rolling/builder.rs

```diff
@@ -181,6 +183,33 @@ impl Builder {
         Self { suffix, ..self }
     }
 
+    /// Keep the last `n` log entries on disk.
```

Comment:
nit: i would probably say "log files" rather than "log entries", since "log entries" seems like it could also refer to an individual log line...
```suggestion
    /// Keep the last `n` log files on disk.
```